app/views/users/terms.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= content_for :breadcrumbs do %>
+  <% if current_user? %>
+    <%= breadcrumbs @user %>
+  <% else %>
+    <%= header_breadcrumbs %>
+  <% end %>
+<% end %>
+
+<div class="mu-user-header">
+  <h1><%= t(:terms_and_conditions) %></h1>
+</div>
+
+<div>
+  <% @profile_terms.each do |term| %>
+    <%= render partial: 'term', locals: {term: term, collapsed: current_user&.has_accepted?(term)} %>
```

Comment:
If user is here, she should be never be nil, right?

Reply:
It could because terms are meant to be shown even for no user. This terms will replace our current static terms altogether